runtime/src/main_launcher.c

```diff
@@ -1,694 +1,26 @@
 /*
  * Copyright 2004-2019 Cray Inc.
  * Other additional copyright holders may be indicated within.
- * 
+ *
```

Comment:
Don't bother fixing these trailing spaces; they're added automatically and will just come back.  Changing them only leads to superfluous content in the diffs.

Reply:
I have no idea what is removing them, to be quite honest.